### PR TITLE
executor, util: refine row.GetMyDecimal

### PR DESCRIPTION
**Pull request by XuHuaiyu**
Before this commit, the result of the following sql is wrong:
``` sql
create table t(a int, b decimal(15,2));
insert into t values(1,771.64),(2,378.49),(3,920.92),(4,113.97);
select a, max(b) from t group by a limit 2;

> 
+------+--------+
| a    | max(b) |
+------+--------+
|    1 | 920.92 |
|    2 | 113.97 |
+------+--------+
```

PTAL @zz-jason @coocood @winoros

### CODE REVIEW

**Comment by zz-jason** [comment]
Good Job !

File: executor/executor.go [hunk 1]
```diff
@@ -510,6 +510,7 @@ func (e *LimitExec) NextChunk(ctx context.Context, chk *chunk.Chunk) error {
 		}
 		e.cursor += batchSize
 	}
+	log.Warning("here")
```
Comment (zz-jason) [inline executor/executor.go]: remove log message.

**Comment by zz-jason** [comment]
LGTM

**Comment by zz-jason** [comment]
@winoros  PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**